test/cases/030_security/010_ports/test.yml

```diff
@@ -0,0 +1,23 @@
+kernel:
+  image: "linuxkit/kernel:4.9.x"
+  cmdline: "console=ttyS0 page_poison=1"
+init:
+  - linuxkit/init:1b8a7e394d2ec2f1fdb4d67645829d1b5bdca037
+  - linuxkit/runc:3a4e6cbf15470f62501b019b55e1caac5ee7689f
+  - linuxkit/containerd:aa8549c2ea757b3daa727bc5a4f484c41952fc12
```

Comment:
this test is failing because this image doesn't exist and isn't signed, should be `linuxkit/containerd:e3832e460e0565d94141a59bc750e6d6523b271f`